samples/dockerbuilds/ARM/Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM ubuntu:latest
 ARG DEBIAN_FRONTEND=noninteractive
+ARG TARGET_BRANCH=main
```

Comment:
Should this be the source branch?

Reply:
Looks like --build-arg will override: https://docs.docker.com/build/guide/build-args/

Reply:
Yes. This is the default value, "main". Its value get set by the pipeline job in .vsts-ci.yml